src/shared/Database/DatabaseSqlite.cpp

```diff
@@ -35,13 +35,15 @@ size_t DatabaseSqlite::db_count = 0;
 DatabaseSqlite::DatabaseSqlite()
 {
     // before first connection
-    if (db_count++ == 0);
+    if (db_count++ == 0)
+    ;
```

Comment:
This was probably inspired by the MySQL class that needed to initialize the MySQL library. As far as I remember SQLite doesn't need anything like that, so I would just drop this from the commit.

And then clean up and remove the whole body (and similar for the destructor) and the ```db_count``` member,  because it's not looking like it serves any purpose anyway...

Reply:
True, I never saw any place aside from the constructor and destructor that access this variable either, I just kept it in case there was something I was missing. I'll remove it